charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -40,6 +40,7 @@ helm.sh/chart: {{ include "gha-runner-scale-set.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/part-of: gha-runner-scale-set-controller
```

Comment:
```suggestion
app.kubernetes.io/part-of: gha-runner-scale-set
```

Reply:
I am using the `app.kubernetes.io/part-of: gha-runner-scale-set-controller` to find the controller deployment during delay role binding.

Reply:
Oh I see, this is my mistake